codalab/worker_manager/worker_manager.py

```diff
@@ -160,8 +151,18 @@ def run_one_iteration(self):
                         len(worker_jobs), self.args.max_workers
                     )
                 )
-                return
+                want_workers = False
+
+        # We have fewer than min_workers, so launch one regardless of other constraints
+        if len(worker_jobs) < self.args.min_workers:
```

Comment:
Is this override also intended for the time constraint? The new added comment on line 136 sounds like we only wanted to override that situation before. I understand that the max worker constraint won't be override assuming args.min_workers < args.max_workers